### PR TITLE
chore: improve the quality of diffs

### PR DESCRIPTION
**Pull request by Gankra**

### CODE REVIEW

**Comment by Gankra** [comment]
image

File: Cargo.lock [hunk 1]
```diff
@@ -319,6 +319,7 @@ dependencies = [
  "serde",
  "serde_json",
  "sha2",
+ "similar",
```
Comment (mistydemeo) [inline Cargo.lock:322]: TIL this was already in our dependency tree. Only via `insta` though so I guess it wasn't in release builds.

**Review by mistydemeo — APPROVED** [review]
Nice! Really like this improved output.

File: Cargo.lock (hunk 1 above)
Comment (Gankra) [reply Cargo.lock:322]: yeah i was just like "ok what the fuck does insta use" and the answer was "a library they made for insta and gloriously spun off as a really nice standalone project"

**Pull request merged**